flink-python/pyflink/fn_execution/coders.py

```diff
@@ -332,6 +400,39 @@ def __init__(self, precision, timezone):
     def get_impl(self):
         return coder_impl.LocalZonedTimestampCoderImpl(self.precision, self.timezone)
 
+    def get_slow_impl(self):
+        return coder_impl_slow.LocalZonedTimestampCoderImpl(self.precision, self.timezone)
+
+
+class PickledBytesCoder(FieldCoder):
+
+    def get_impl(self):
+        return None
+
+    def get_slow_impl(self):
+        return coder_impl_slow.PickledBytesCoderImpl()
+
+
+class TupleCoder(FieldCoder):
+
+    def __init__(self, field_coders):
+        self._field_coders = field_coders
+
+    def get_slow_impl(self):
+        return self._create_impl()
+
+    def get_impl(self):
+        return None
+
+    def _create_impl(self):
+        return coder_impl_slow.TupleCoderImpl([c.get_impl() for c in self._field_coders])
```

Comment:
Maybe it is `return coder_impl_slow.TupleCoderImpl([c. get_slow_impl() for c in self._field_coders])` ?